find-phantom-resource.ps1

```
@@ -354,91 +369,125 @@ if ($SubScan) {
+            $deletingRGs.Add($rg)
         }
-        $rgData = Invoke-AzRest -Url "/subscriptions/$subscriptionId/resourceGroups/$RgName`?api-version=2021-04-01"
-        if ($null -eq $rgData) {
-            $rgStatusCache[$key] = "NotFound"
-            return "NotFound"
+
+        # Direct REST scan - same reliable method as single-RG mode.
+        # Bypasses JMESPath casing bug and finds phantoms invisible to az resource list.
+        $restResult = Invoke-AzRest -Url "/subscriptions/$subscriptionId/resourceGroups/$rgName/resources?api-version=2024-03-01"
+        if ($null -eq $restResult -or $null -eq $restResult.value) {
+            Write-Host " (REST failed, skipped)"
```

PSScriptAnalyzer (1.24.0)

[warning] File 'find-phantom-resource.ps1' uses Write-Host. Avoid using Write-Host because it might not work in all hosts, does not work when there is no host, and (prior to PS 5.0) cannot be suppressed, captured, or redirected. Instead, use Write-Output, Write-Verbose, or Write-Information.

(PSAvoidUsingWriteHost)